package.json

```diff
@@ -1,9 +1,8 @@
 {
   "name": "@cotecna/stencil-components",
-  "version": "0.0.1-beta.1",
+  "version": "0.0.1-alpha.5",
```

Comment:
Do we need to publish an alpha version? Or maybe we need to create a stable one, such as: 0.0.1